### DISCUSSION TITLE
Extensibility Open Session - July 28

### DISCUSSION THREAD

**Discussion by stockholmux** [runfinch/finch#479] [General]
Hey folks!

We're setting up an open session over zoom to get feedback on Finch extensions on July 28 at 9:30AM Pacific time.

More details on the meeting as they come in. Let me know if you have any questions!

-Kyle from AWS

**Comment by kcq** [comment]
can somebody share the HackMD notes page link from the meeting?

**Reply by stockholmux** [reply]
Gladly: https://hackmd.io/W7CgOeUtRdys_xmKRl_x4A

**Comment by kcq** [comment]
Not sure if you have that internally, but it's useful to have one list that captures the extension models in relevant tools/products (e.g., kubectl plugins, docker cli plugins, etc) and another list that captures different extension implementation options (e.g., process shell out and redirecting terminal io, process shell out and using standard in/out as the IPC channel, the HashiCorp gRPC extensions, WASM extensions, etc).
